doc/howtos/booting-and-installation.rst

```diff
@@ -131,13 +131,12 @@ Flashing an Intel Edison requires use of a breakout board and two micro-USB cabl
    for recovery cases.)
 #. Plug in a micro-USB cable to the J3 connector on the board (corner next to the FTDI chip)
 #. Flip the DIP switch towards jumper J16
-#. Open :command:`minicom` or other terminal program on your host computer to attach to the serial console
-#. Download the ``flashall`` folder from the Ostro OS download folder for edison (on https://download.ostroproject.org)
-#. Copy the flashall script (``flashall.sh``) from the flashall folder to the Ostro OS image folder
-#. Then in the image folder run:: 
-
-    $ sudo ./flashall.sh
-
+#. Download the ``ostro-image`` or ``ostro-image-dev`` image from the Ostro OS download folder for edison (on https://download.ostroproject.org/releases/ostro-os/milestone/)
+#. Extract the image using the command ``$ tar xf ostro-image-*-edison-*.toflash.tar.bz2`` archive
+#. Change directory to the toFlash folder
+#. Run the command ``$ sudo ./flashall.sh`` |br|
+   **NOTE:** To flash the ``-dev`` image please use the ``-i <image-name>``
+   command line argument 
```

Comment:
@barbieri, please remember. Making the script smart means another tool to update if the image naming convention changes. It also means another thing for QA to test. This is why I prefer appending the image name for all images types. It's clean to document and it doesn't require updating the code with naming convention changes.